### DISCUSSION TITLE
Enhanced RTMP V2 (alpha)

### DISCUSSION THREAD

**Discussion by veovera** [veovera/enhanced-rtmp#26] [Announcements]
We are excited to announce the release of the latest alpha specification, "[enhanced-rtmp-v2](https://veovera.github.io/enhanced-rtmp/docs/enhanced/enhanced-rtmp-v2)" (or "[enhanced-rtmp-v2.pdf](https://veovera.github.io/enhanced-rtmp/docs/enhanced/enhanced-rtmp-v2.pdf)"). This pivotal publication signifies our ongoing commitment to the advancement of the Real-Time Messaging Protocol (RTMP). The "enhanced RTMP" initiative introduces several critical enhancements, including:

- **Integration of Advanced Video Codecs:** Incorporating VP8, VP9, HEVC, and AV1 with High Dynamic Range (HDR) support to meet the needs of modern displays and video content requirements.
- **Addition of VideoPacketType.Metadata:** This expansion broadens the range of supported video metadata types, enriching the streaming experience.
- **Introduction of Video Multitrack Capabilities:** Enabling the concurrent management and processing of multiple video streams, this feature fosters a more versatile streaming infrastructure.
- **Implementation of a Reconnect Request Feature:** Focused on improving connection stability and resilience, ensuring a more reliable streaming experience.

For detailed information on all advancements, please refer to the documents linked above.

Best regards,

Team Veovera

**Comment by jordicenzano** [comment]
Thanks for working on that! 

Just a thought It would be great to add few examples to the doc I think that could help to understand the changes, for instance:
- Example of RTMP connection using VP9
- Example of RTMP connection using HEVC and HDR
- Example of video multitrack, simple 2 lanes h264 (could be 360p 500Kbps  h264, and 720p 2000Kbps  h264)
- Example of video multitrack, simple 2 lanes h264 (could be 360p 500Kbps  h264, and 720p 2000Kbps  h264)
- Example of reconnection

**Reply by veovera** [reply]
Hey Jordi, great suggestion. We will see what we can provide. Perhaps real world examples would be best.

thx
